Bottom: 7b14bf4de6e1cc019909ac9ddea4bb25ec59fbea
Top:    ef09ec66b6b08cc55638b7a11eea6e59aaecb257
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-18 23:05:38 +0100

Refresh of clarity-temp

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index c5f3e3e..2a0cd9e 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -2433,7 +2433,6 @@ idle_proc (XtPointer dummy)
       int mx, my;
       BoxType region;
       lesstif_use_mask (0);
-      Crosshair.On = 0;
       pixmap = main_pixmap;
       mx = view_width;
       my = view_height;
@@ -2519,6 +2518,7 @@ idle_proc (XtPointer dummy)
       XCopyArea (display, main_pixmap, window, my_gc, 0, 0, view_width,
 		 view_height, 0, 0);
       pixmap = window;
+      Crosshair.On = 0;
       CrosshairOn ();
       need_redraw = 0;
     }
